src/main/webapp/reviewCategories/ReviewCategories.js

```diff
@@ -1,34 +1,81 @@
 import React, { Component } from 'react';
-import { h, div, span, a, h1 } from 'react-hyperscript-helpers';
+import { h, h1, div, a, span } from 'react-hyperscript-helpers';
+import { Reports } from "../util/ajax";
 import { spinnerService } from "../util/spinner-service";
-import { Spinner } from '../components/Spinner';
-import { Report } from '../util/ajax';
+import { Spinner } from "../components/Spinner";
+import { TableComponent } from "../components/TableComponent";
+import { CATEGORY_SORT_NAME_INDEX, STYLES_REVIEW_CATEGORIES } from "../util/ReportConstants";
+import { TABLE_ACTIONS } from "../util/TableUtil";
 import { handleRedirectToProject } from "../util/Utils";
-import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
-import './reviewCategories.css';
 
-const styles = { 
-  projectWidth: '22',
-  summaryWidth: '130',
-  statusWidth: '30',
-  reviewWidth: '42'
-};
+const SIZE_PER_PAGE_LIST = [
+  { text: '50', value: 50 },
+  { text: '100', value: 100 },
+  { text: '500', value: 500 }];
 
-const headers =
-  [
-    { name: 'Project', value: 'projectKey' },
-    { name: 'Summary', value: 'summary' },
-    { name: 'Status', value: 'status' },
-    { name: 'Review Category', value: 'reviewCategory' }
-  ];
+const columns = [
+  {
+    dataField: 'id',
+    text: 'id',
+    hidden: true,
+    csvExport : false
+  } ,
+  {
+    dataField: 'projectKey',
+    text: 'Project',
+    sort: true,
+    headerStyle: (column, colIndex) => {
+      return { width: STYLES_REVIEW_CATEGORIES.projectKeyWidth};
+    },
+    formatter: (cell, row, rowIndex, colIndex) =>
+    div({},[
+      a({ href: handleRedirectToProject(component.serverURL, row.projectKey) },[row.projectKey])
+    ]),
+  },
+  {
+    dataField: 'summary',
+    text: 'Summary',
+    sort: true,
+    headerStyle: (column, colIndex) => {
```

Comment:
It seems like `column, colIndex` values are not in use here. Are those default values of this type of functions?